hw/ip/otbn/dv/otbnsim/sim/insn.py

```diff
@@ -886,11 +886,11 @@ def execute(self, state: OTBNState) -> None:
         state.wdrs.get_reg(wrd).write_unsigned(value)
 
         if self.grd_inc:
-            new_grd_val = (grd_val + 1) & ((1 << 32) - 1)
+            new_grd_val = (grd_val + 1) & 31
```

Comment:
I'd tend towards using a hex constant here `0x1f`, in my mind at least easier to realize that represents a run of set bits. Python does support binary literals so could even use `0b11111`

Reply:
It also turns out to be how I wrote it for bn.sid/bn.lid. Thanks for spotting it.